doc/exclude_patterns.inc

```diff
@@ -0,0 +1,7 @@
+# The following patterns are to be excluded from the documentation build
+examples
+tests
+.pytest_cache
+CHANGELOG.rst
+LICENSE.rst
+README.md
```

Comment:
Not important, but not sure how README.md snuck in here. Can't see the file?

Reply:
Got copied from another repo. I'll remove it.